site/src/pages/RoadmapPage/sidebar/GERequiredCourseList.tsx

```diff
@@ -4,9 +4,25 @@ import trpc from '../../../trpc';
 import RequirementsLoadingIcon from './RequirementsLoadingIcon';
 import { setGERequirements } from '../../../store/slices/courseRequirementsSlice';
 import { useAppDispatch, useAppSelector } from '../../../store/hooks';
+import { findCommonLabelPrefix } from '../../../helpers/substitutions';
 
-function getCoursesForGE() {
-  return trpc.programs.getRequiredCoursesUgrad.query({ id: 'GE' });
+async function getCoursesForGE() {
+  const fetchedCourses = await trpc.programs.getRequiredCoursesUgrad.query({ id: 'GE' });
+
+  /* if a top level "Select 1 of the following" group of requirements is seen, attempt to replace the label */
+  fetchedCourses.forEach((geRequirement) => {
+    if (geRequirement.requirementType === 'Group' && geRequirement.label === 'Select 1 of the following') {
+      const requirementSubLabels = geRequirement.requirements.map((req) => req.label);
```

Comment:
nitpick on naming – can be something more like `nestedLabels` since 'sub' can be ambiguous and 'requirement' is redundant